### ISSUE TITLE
Embed files does not work

### ISSUE THREAD

**Issue by gigaga** [docsifyjs/docsify#1005]
## Bug Report
By using the last release of docsify (from CDN https://cdn.jsdelivr.net/npm/docsify@4), the feature Embed files does not work.

#### Steps to reproduce

From the code below :
```
[filename](./test.md ":include")
```

#### What is current behaviour
The render will be a link name filename.

#### What is the expected behaviour
The test.md content should be rendered directly (embedded).

#### Other relevant information

- [ ] Bug does still occur when all/other plugins are disabled?

- Your OS: Windows
- Node.js version: v10.15.3
- npm/yarn version: npm 6.4.1
- Browser version: Chrome
- Docsify version: 4.4.0
- Docsify plugins: tried without any plugins, but I have the same issue

**Comment by anikethsaha** [comment]
is it same for prev version ?

**Comment by gigaga** [comment]
I tried with : 
- https://cdn.jsdelivr.net/npm/docsify@4.10.2 --> KO
- https://cdn.jsdelivr.net/npm/docsify@4.6.0 --> It works
- https://cdn.jsdelivr.net/npm/docsify@4.8.0 --> It works
- https://cdn.jsdelivr.net/npm/docsify@4.9.0 --> KO

:(

**Comment by anikethsaha** [comment]
thanks, I will check it soon 👍

**Comment by gigaga** [comment]
Also, from @4.8.0, embed file feature works, however, from a specific plugin, I'd like to change content so, from 

 ```javascript
$docsify.plugins = [].concat(install, $docsify.plugins);

// Install a specific plugin to force current sidebar item active into collapse state
function install(hook, vm) {
  hook.beforeEach(function(content) {
    // Invoked each time before parsing the Markdown file.
    // ...
    console.log(content)
    return content;
  });
}
```

`content` is only the content of main MD file. I'm not able to change content of an embed file :(.

Thanks a lot for your feedback!

**Comment by anikethsaha** [comment]
Yes it can't read the embedded file's code

**Comment by gigaga** [comment]
No workaround is possible?
Thanks

**Comment by anikethsaha** [comment]
i cant say that right now, I need to check and try.
I am not sure how markedjs process embedded files that's why. 
Feel free to investigate 👍

**Comment by stratosgear** [comment]
what a coincidence! I just got hit by it right now!

Try with single quotes, like in: `[filename](./test.md ":include")`.

Works for me!

**Comment by anikethsaha** [comment]
> what a coincidence! I just got hit by it right now!
> 
> Try with single quotes, like in: `[filename](./test.md ':include')`.
> 
> Works for me!

@gigaga can you try this

**Comment by gigaga** [comment]
Hello @anikethsaha 

Like you, it's work by using single quotes!

Thanks!

**Comment by gigaga** [comment]
About embedded files processing, I can do what I want by override markdown function.

Thanks again :)

**Comment by anikethsaha** [comment]
Yea you can use the plugin hook to get into the core op.

**Comment by trusktr** [comment]
Will be fixed by #1598
